lib/plug/mailbox_preview.ex

```diff
@@ -22,7 +22,7 @@ if Code.ensure_loaded?(Plug) do
     use Plug.Router
     use Plug.ErrorHandler
 
-    alias Swoosh.InMemoryMailbox
+    alias Swoosh.{Email.Format, InMemoryMailbox}
```

Comment:
Personally dislike the look of this, but understand it's shorter and likely "idiomatic".